### ISSUE TITLE
The ModalWrapper doesn't have a keyboard trap

### ISSUE THREAD

**Issue by zgrossbart** [IBM/carbon-components-react#874]
1. Go here:  http://react.carbondesignsystem.com/?selectedKind=ModalWrapper&selectedStory=transactional%20modal&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

2. Click the *Transactional Modal* button to open the dialog.

3. Press the tab key.  At this point the focus will be on the close button in the upper right.

4. Press the tab key twice.  At this point the focus will be on the Save button.

5. Now press the tab again.  The focus will leave the dialog and end up on the *Show Info* button.

If you keep pressing the tab key the focus will leave the modal and end up on different items in the page.  For accessibility it should stay within the modal dialog until it is dismissed.

**Comment by zgrossbart** [comment]
There is also a Jira issue to track this problem.  cloudlab-jira.canlab.ibm.com:8080/browse/UI-1510

**Comment by asudoh** [comment]
Thank you for repoting @zgrossbart! Having contributions porting [`carbon-components/src/components/modal/modal.js#L140-L153`](https://github.com/carbon-design-system/carbon-components/blob/3e9bc26/src/components/modal/modal.js#L140-L153) to this React repo would be nice.

**Comment by aledavila** [comment]
Closing fixed in above.
